Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -1396,7 +1396,6 @@ void BPReload()
   // restore anything that goes straight to the renderer.
   // let's not risk actually replaying any writes.
   // note that PixelShaderManager is already covered since it has its own DoState.
-  SetGenerationMode();
```

Comment:
Is there a reason to drop this function here?

Reply:
Nope, that looks like a mistake on my part.